content/docs/1.1.0/high-availability/node-failure.md

```diff
@@ -55,6 +55,22 @@ Which then allows the pending replacement pods to start correctly with the reque
 Longhorn will recover the Volume Attachment from a failed node as soon as there are pending replacement pods available.
 Which then allows the pending replacement pods to start correctly with the requested volumes being available.
 
+### Pod Deletion Policy When Node Downs
+
+Kubernetes never force deletes pods of StatefulSet or Deployment on a downed node. Since the pod on the down node wasn't removed,
+the volume will be stuck on the down node with it as well. The replacement pods cannot be started because the Longhorn volume is
+RWO (see more about access modes [here](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes) ), which can
+only be attached to one node at a time. Longhorn provides an option for users to help automatically force delete terminating pods
+of StatefulSet/Deployment on the downed node. After force deleting, Kubernetes will detach Longhorn volume and spin up replacement
+pods on a new node. Specifically, Longhorn provides 4 different Pod deletion policies when node downs:
+1. `do-nothing` is the default Kubernetes behavior of never force deleting StatefulSet/Deployment terminating pods on the downed node.
+1. `delete-statefulset-pod` Longhorn will force delete StatefulSet terminating pods which use Longhorn volumes on the downed node.
+1. `delete-deployment-pod` Longhorn will force delete Deployment terminating pods which use Longhorn volumes on the downed node.
+1. `delete-both-statefulset-and-deployment-pod` Longhorn will force delete StatefulSet/Deployment terminating pods which use Longhorn
+volumes on the downed node.
+
+You can find the `Pod Deletion Policy When Node Downs` in setting tab in Longhorn UI or [Customizing Default Settings](../advanced-resources/deploy/customizing-default-settings/)
```

Comment:
> [Customizing Default Settings](../advanced-resources/deploy/customizing-default-settings/)
@shuo-wu I don't know how to make this link to work. Can you help me?

Reply:
It should be `../../advanced-resources/deploy/customizing-default-settings`

Reply:
Thank you!